lib/service/postcard_service.dart

```diff
@@ -362,4 +365,41 @@ class PostcardServiceImpl extends PostcardService {
   Future<void> cancelSharePostcard(AssetToken asset) async {
     await sharePostcard(asset);
   }
+
+  @override
+  Future<PostcardLeaderboard> fetchPostcardLeaderboard() async {
+    final leaderboardResponse =
```

Comment:
I think should check the `DistanceFormatter` or the `LocaleService` to get the right DistanceUnit.